dist/data/cloudinary.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
```

Comment:
hey looks like you accidentally checked in `dist` before you added it to .gitignore

if you're not familiar, you can:
- remove from gitignore, remove the directory, commit the changes
- add a separate new commit with it in gitignoree